src/Shared/LogMessagePacketBase.cs

```diff
@@ -487,17 +487,13 @@ private static Delegate CreateDelegateRobust(Type type, Object firstArgument, Me
                     delegateMethod = methodInfo.CreateDelegate(type, firstArgument);
 #endif
                 }
-                catch (FileLoadException)
+                catch (FileLoadException) when (i < 5)
```

Comment:
`i != 5` is not equivalent of `i < 5`. I like this change as it is, if I understand it correctly, safer against infinite loop.
I don't think of any negative consequences of the code as is, do you?

Reply:
Yeah; I think it's good.